From 3f06b21b53c2691480c8cc4ba190798c8b2e7dc5 Mon Sep 17 00:00:00 2001
From: Eugene Syromiatnikov <esyr@redhat.com>
Date: Tue, 22 Aug 2023 17:34:43 +0200
Subject: [PATCH 08/10] releasenote.md: remove the duplicating 06-9e-0c/22
 record

CFL-S stepping P0 (06-9e-0c/22) is already listed as "CFL-H/S"
and it has not been listed in the notes to the previous releases
separately.

* releasenote.md (microcode-20230808) <CFL-S P0>: Remove.

Signed-off-by: Eugene Syromiatnikov <esyr@redhat.com>
---
 releasenote.md | 1 -
 1 file changed, 1 deletion(-)

diff --git a/releasenote.md b/releasenote.md
index e86759f..5847f3c 100644
--- a/releasenote.md
+++ b/releasenote.md
@@ -49,7 +49,6 @@
 | CFL-H/S/E3     | U0       | 06-9e-0a/22 | 000000f2 | 000000f4 | Core Gen8 Desktop, Mobile, Xeon E
 | CFL-H/S        | P0       | 06-9e-0c/22 | 000000f2 | 000000f4 | Core Gen9
 | CFL-S          | B0       | 06-9e-0b/02 | 000000f2 | 000000f4 | Core Gen8
-| CFL-S          | P0       | 06-9e-0c/22 | 000000f2 | 000000f4 | Core Gen9 Desktop
 | CFL-U43e       | D0       | 06-8e-0a/c0 | 000000f2 | 000000f4 | Core Gen8 Mobile
 | CLX-SP         | B0       | 06-55-06/bf | 04003501 | 04003604 | Xeon Scalable Gen2
 | CLX-SP         | B1       | 06-55-07/bf | 05003501 | 05003604 | Xeon Scalable Gen2
-- 
2.13.6

